spec/syntax.md

```diff
@@ -575,39 +624,40 @@ private-start = "&" / "^"
 > Protect stuff in {^ph}<a>{^/ph}private use{^ph}</a>{^/ph}
 > ```
 
-#### Reserved
+#### Reserved Annotations
 
-A **_<dfn>reserved</dfn>_** _annotation_ is an _annotation_ whose syntax is reserved
+A **_<dfn>reserved annotation</dfn>_** is an _annotation_ whose syntax is reserved
 for future standardization.
 
-A _reserved_ _annotation_ starts with a reserved character.
-A _reserved_ _annotation_ MAY be empty or contain arbitrary text after its first character.
+A _reserved annotation_ starts with a reserved character.
+A _reserved annotation_ MAY be empty or contain arbitrary text after its first character.
 
 This allows maximum flexibility in future standardization,
 as future definitions MAY define additional semantics and constraints
 on the contents of these _annotations_.
-A _reserved_ _annotation_ does not include trailing whitespace.
+A _reserved annotation_ does not include trailing whitespace.
 
 Implementations MUST NOT assign meaning or semantics to
 an _annotation_ starting with `reserved-start`:
 these are reserved for future standardization.
-Implementations MUST NOT remove or alter the contents of a _reserved_ _annotation_.
+Implementations MUST NOT remove or alter the contents of a _reserved annotation_.
 
 While a reserved sequence is technically "well-formed",
 unrecognized reserved sequences have no meaning and MAY result in errors during formatting.
```

Comment:
```suggestion
unrecognized reserved sequences have no meaning and MUST result in errors during formatting.
```

Reply:
The normative language MAY not be appropriate here 😁. Also, "unrecognized reserved sequence" is an oxymoron. If it were recognized, it would not be reserved. 

Since we eliminated reserved from complex-body, a reserved sequence is not in the way of selecting the pattern. The question we have to ask ourselves (which is what @gibson042's comment is really about...) is: is a reserved sequence an error or default ignorable? If it is an error, it not only "MUST result in errors during formatting", it needs to result in a *specific* error (Unsupported Statement?). If it is not an error, it "MUST NOT result in an error during formatting". 

Which behavior do we prefer?

Reply:
We would not error on unselected patterns (per your quote). Any MUST here would only apply inside a pattern being formatted. Note that reserved-statement and reserved-annotation are different things. @gibson042's suggestion should be specific about which it means. I think the right choice here is actually to stop sooner. Note that the front of the sentence says: 
> While a reserved sequence is technically "well-formed",

So we should just finish up with:

```suggestion
unrecognized _reserved-annotations_ or _private-use-annotations_ have no meaning.
```

This is the syntax spec and the normative text is really a "forward reference" to error handling in formatting.md. We should just let formatting do the work. Or we could say that unrecognized placeholders turn into fallbacks during formatting.

Reply:
Works for me too.